docs/README.md

```diff
@@ -0,0 +1,325 @@
+EVE is based on a type-1 hypervisor and it does not directly support
+traditional POSIX-like application and processes. All applications
+running on EVE have to be represented as [Edge Containers](ECOS.md) which
+can be either Virtual Machines, Containers or Unikernels. EVE presents a
+somewhat of a novel design of an operating systems, but at the same time
+it is heavily inspired by ideas from [Qubes OS](https://www.qubes-os.org/),
+[ChromeOS](https://www.chromium.org/chromium-os/chromiumos-design-docs),
+[Core OS](https://coreos.com/) and [Smart OS](https://wiki.smartos.org/).
+
+To put it another way, EVE is all you need to put on your Edge Nodes in order
+to run Edge Containers there. You don't need to install traditional
+(or embedded) operating systems, hypervisors, container engines and middle
+ware. EVE provides all of that in a tightly integrated, secure and space
+conscious package.
+
+EVE's promise to its users is simple:
+* Zero touch: you should never have physical access to your Edge Node,
+  after installing EVE on it. All hardware and software management should
+  be safely performed remotely
+* Zero trust: not a single individual component could be fully trusted
+  and the only way to build a trustworthy system is by utilizing
+  defence-in-depth and state of the art cryptography primitives
+* EVE will make *any kind of* application type, network environment and
+  hardware be easily and safely manageable
+* EVE promises to bring cloud-native, DevOps-centric principles of software
+  development to Edge Computing
+
+This documentation is a work-in-progress tracking the current state of EVE.
+It is expected to evolve significantly based on community feedback and constant
+addition of new functionality to EVE. The structure of this [README](README.md)
+is aimed at providing just enough information to vector you off to a relevant
+part  of the documentation. Even then, it can be too long for some readers and
+we  provide a companion [FAQ](FAQ.md) for those simply trying to look random
+facts  up. Finally, if you are interested in the gory details of how each
+subcomponent of EVE is implemented you should refer to respective subcomponents
+[docs folders](../pkg/) and especially [pillar docs](../pkg/pillar/docs/README.md).
+
+EVE leverages an enormous amount of open source projects and it would *not* have
+been possible to build EVE without relying on them. When it comes to developing
+EVE we practice a strict [Upstream First](UPSTREAMING.md) policy.
+
+## Edge Containers
+
+Edge Container is a novel concept that allows an effective packaging and lifecycle
+management of the following types of workloads:
+
+* [Traditional Virtual Machines](https://en.wikipedia.org/wiki/Virtual_machine)
+* [Unikernels](http://unikernel.org/)
+* [Docker/OCI container](https://www.opencontainers.org)
+
+EVE is *not* a traditional operating system in that it doesn't support applications
```

Comment:
"doesn't support running workload applications" - it isn't that POSIX is disabled or it is something else, as much as we prevent it.

Reply:
fixed